src/Tizen.System/Device/PerformanceControlType.cs

```diff
@@ -0,0 +1,80 @@
+/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+using Tizen.Common;
+using System.ComponentModel;
+
+namespace Tizen.System
+{
+    /// <summary>
+    /// This class represents the type of event for backends. This class can be converted from the string type.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class PerformanceControlType
```

Comment:
Why not enum?

Reply:
At this moment we decide to support only two request types(AppLaunchHome, HomeScreen), but kernel provide  many pmqos request types. In the future, there are possibilities to adding more request types into C# internal API.
Due to constraint of enum type, if new request type has added & if order has changed, value also changed,
If enum values are different with previous API level, it will broken compatibility.
Mr. WonyoungChoi help me to solve this limitation.